src/docs/testing/common-errors.md

```diff
@@ -0,0 +1,451 @@
+---
+title: Common Flutter Errors
+description: How to recognize and resolve common Flutter Framework errors.
+---
+
+## Introduction
+
+This page explains several frequently-encountered Flutter Framework errors and gives suggestions on how to resolve them. This is a living document with more errors we’d like to add in future revisions, and we welcome contributions from all Flutter users. Feel free to open an issue or submit a pull request to make this page more useful to you and the Flutter community. 
+
+
+### List of Errors
+
+* [A RenderFlex overflowed…](#a-renderflex-overflowed)
+* [RenderBox was not laid out](#renderbox-was-not-laid-out)
+* [Vertical viewport was given unbounded height](#vertical-viewport-was-given-unbounded-height)
+* [Incorrect use of ParentData widget](#incorrect-use-of-parentdata-widget)
+* [‘An InputDecorator...cannot have an unbounded width’](#an-inputdecoratorcannot-have-an-unbounded-width)
+* [setState called during build](#setstate-called-during-build)
+
+
+
+## ‘A RenderFlex overflowed…’
+
+
+### What does the error look like?
+
+RenderFlex overflow is one of the most frequent Flutter Framework errors we’ve seen, and you probably have run into it already. When it happens, you’ll see yellow & black stripes indicating the area of overflow in the app UI in addition to the error message in the debug console: 
+
+<!-- ![alt_text](images/image1.png "image_tooltip") -->
```

Comment:
I'm guessing these images are coming at some point?

Reply:
I'll just use text to keep things simple and searchable.